modules/imaging_browser/jsx/imagingBrowserIndex.js

```diff
@@ -0,0 +1,199 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import {Tabs, TabPane} from 'Tabs';
+import Loader from 'Loader';
+import FilterableDataTable from 'FilterableDataTable';
+
+class ImagingBrowserIndex extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            data: {},
+            error: false,
+            isLoaded: false,
+        };
+
+        this.fetchData = this.fetchData.bind(this);
+        this.formatColumn = this.formatColumn.bind(this);
+    }
+
+    componentDidMount() {
+        this.fetchData()
+            .then(() => this.setState({isLoaded: true}));
+    }
+
+    /**
+     * Retrieve data from the provided URL and save it in state
+     * Additionally add hiddenHeaders to global loris variable
+     * for easy access by columnFormatter.
+     *
+     * @return {object}
+     */
+    fetchData() {
+        return fetch(this.props.dataURL, {credentials: 'same-origin'})
+            .then((resp) => resp.json())
+            .then((data) => this.setState({data}))
+            .catch((error) => {
+                this.setState({error: true});
+                console.error(error);
+            });
+    }
+
+    /**
+     * Modify behaviour of specified column cells in the Data Table component
+     *
+     * @param {string} column - column name
+     * @param {string} cell - cell content
+     * @param {object} row - row content indexed by column
+     *
+     * @return {*} a formated table cell for a given column
+     */
+    formatColumn(column, cell, row) {
+        // Set class to 'bg-danger' if file is hidden.
+        const style = '';
+        let result = <td className={style}>{cell}</td>;
+        switch (column) {
+            case 'New Data':
+                if (cell === 'new') {
+                    result = (
```

Comment:
You can condense this to one line since it's not over 80 characters
```suggestion
                    result = <td className="newdata">NEW</td>;
```